### PR TITLE
tape instead mocha

### PR DESCRIPTION
**Pull request by fanatid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,36 +13,35 @@
...
   },
   "dependencies": {
     "bs58": "^2.0.1",
     "create-hash": "^1.1.0"
   },
   "devDependencies": {
-    "blanket": "*",
-    "mocha": "*",
-    "standard": "*",
-    "travis-cov": "*"
```
Comment (dcousens) [inline package.json]: What was wrong with `travis-cov`?

File: package.json [hunk 2]
```diff
@@ -23,36 +13,35 @@
...
+    "coveralls": "^2.11.9",
+    "nyc": "^6.1.1",
```
Comment (dcousens) [inline package.json]: `nyc`... ugh. 
Can we just stick with `blanket`?
I don't like waiting an hour to install all my dependencies.

Comment (fanatid) [inline package.json]:
```
$ time npm i standard
real	0m29.576s
$ time npm i nyc
real	0m32.639s
```
time almost same :)

in blanket readme: If you're looking for a more active project for JavaScript code coverage, I recommend Istanbul.

File: package.json (hunk 1 above)
Comment (fanatid) [inline package.json]: are we really need send data to coveralls?

File: package.json (hunk 2 above)
Comment (dcousens) [inline package.json]:
``` bash
time npm i blanket
...
real    0m3.810s
user    0m1.123s
sys     0m0.113s
```

``` bash
time npm i standard
...
real    0m35.819s
user    0m6.267s
sys     0m0.723s
```

``` bash
time npm i nyc
real    0m49.740s
user    0m6.183s
sys     0m0.500s
```

I agree, `standard` takes too long (halp @feross).
This doesn't mean I want to *double* that time with `nyc`.

File: package.json (hunk 1 above)
Comment (fanatid) [inline package.json]: sorry, was wrong about coveralls!

File: package.json [hunk 3]
```diff
@@ -23,36 +13,34 @@
...
   "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bitcoinjs/bs58check.git"
+  },
   "scripts": {
     "standard": "standard",
-    "coverage": "mocha --require blanket -R travis-cov",
-    "coverage-local": "mocha --require blanket -R html-cov",
+    "coverage": "nyc npm run unit",
```
Comment (dcousens) [inline package.json]: How do you set the coverage threshold for travis?

Comment (fanatid) [inline package.json]: I think with flag `--check-coverage`

**Comment by fanatid** [comment]
@dcousens in current master:
```
$ npm run coverage
Error: BLANKET-Error parsing instrumented code: ReferenceError: _$jscoverage is not defined
```

**Comment by dcousens** [comment]
Welp.   Well, can you maybe separate the `tape` change from it for now and put this in another PR?

**Comment by fanatid** [comment]
tape + blanket + travis-cov?

**Comment by dcousens** [comment]
Yeah, for now

**Comment by dcousens** [comment]
Any reason for cloes?

**Comment by fanatid** [comment]
Nope, just not sure that I ready to do this

File: package.json (hunk 2 above)
Comment (feross) [inline package.json]: You're measuring install time? `standard`'s install time is almost all due to ESLint, which has a lot of dependencies.

It's way faster for me, though:

```
npm i standard  8.72s
```

**Comment by fanatid** [comment]
@dcousens can you fix `coverage` command?

**Comment by dcousens** [comment]
@fanatid fixed

**Pull request merged**